addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComHandler.java

```diff
@@ -45,14 +44,11 @@
  * @author Pauli Anttila - Initial contribution
  */
 public class RFXComHandler extends BaseThingHandler implements DeviceMessageListener {
+    private final static int LOW_BATTERY_LEVEL = 1;
 
-    private Logger logger = LoggerFactory.getLogger(RFXComHandler.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(RFXComHandler.class);
```

Comment:
We had long discussions before defining [E.1 in our guidelines](https://www.eclipse.org/smarthome/documentation/development/guidelines.html). The main argument for the decision was the "IOC friendlyness" as we have to consider OSGI DS as an IOC mechanism. So please adapt the code again to match these guidelines - thanks!